poetry.lock

```diff
@@ -1,89 +1,10 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
These are a lot of changes, can we remove this from the diff, and figure out why there's an inconsistency here?

Reply:
Looks like a poetry version difference.